addons/sms_twilio/tests/common.py

```diff
@@ -110,26 +129,58 @@ def _setup_sms_twilio(cls, company):
             "sms_provider": "twilio",
             "sms_twilio_account_sid": "AC12345678987654321234567898765432",
             "sms_twilio_auth_token": "grimgorironhide",
+            "sms_twilio_number_ids": [
+                (5, 0),
+                (0, 0, {
```

Comment:
What are these scriptures?

Reply:
It is like Commands, but not deprecated.